routing/mock/centralized_test.go

```diff
@@ -45,7 +45,7 @@ func TestClientFindProviders(t *testing.T) {
 	providersFromClient := client.FindProvidersAsync(context.Background(), k, max)
 	isInClient := false
 	for pi := range providersFromClient {
-		if pi.ID == pi.ID {
+		if pi.ID == pi.ID { // <-- typo?
```

Comment:
this code is so old and unused... We can probably just remove routing/mock and the supernode routing stuff in general (as a separate PR)